readme/src/main/scala/org/scalafmt/readme/Readme.scala

```diff
@@ -100,19 +101,50 @@ object Readme {
     sideBySide(code, formatted)
   }
 
+  def changedConfig(style: ScalafmtConfig): String = {
+    // Quite a hacky implementation for now.
+    // This only works as we're flattening the HOCON configuration so
+    //   a = [
+    //    b = 1
+    //    c = 2
+    //   ]
+    // turns into
+    //   a.b = 1
+    //   a.c = 2
+    // which means a simple line-by-line string diffing is sufficient for now.
+    val defaultStrs = Config.toHocon(ScalafmtConfig.default).toSet
+    val configStrs = Config.toHocon(style).toSet
+    configStrs.diff(defaultStrs).mkString("\n")
+  }
+
+  def configurationBlock(style: ScalafmtConfig) = {
+    div(
+      span(
+        "Show/hide configuration used for this example",
+        `class` := "scalafmt-configuration-toggle"),
+      pre(changedConfig(style)),
+      `class` := "scalafmt-configuration collapsed"
```

Comment:
Is it possible to have it expanded by default or does that result in too much bloat?